src/Data/Gdpr/LocalStorage/localStorageGdprDataClass.ts

```diff
@@ -6,6 +6,7 @@ async function attributes(): Promise<DataClassSchema> {
   const lang = await load(currentLanguage)
 
   return {
+    active: ['boolean', { title: lang === 'de' ? 'Aktiv' : 'Active' }],
```

Comment:
PisaSales updated this to `Aktiv?`/`Active` in the meantime. Can you adjust all names accordingly?